recipe/build.sh

```diff
@@ -112,6 +112,10 @@ fi
 # Thus: don't add com_google_protobuf here.
 export TF_SYSTEM_LIBS="boringssl,com_github_googlecloudplatform_google_cloud_cpp,com_github_grpc_grpc,flatbuffers,zlib,com_google_absl"
 
+if [[ "${target_platform}" == "osx-64" ]]; then
+    export TF_SYSTEM_LIBS="${TF_SYSTEM_LIBS},onednn"
```

Comment:
Shouldn't we do this on linux-64 as well? 🤔

Reply:
I'm a bit cautious. I know that in the past, in TensorFlow, they have used some special build options. If it works, let's also do it for linux-64.